core/planner.c

```diff
@@ -206,6 +206,16 @@ void fill_default_cylinder(const struct dive *dive, cylinder_t *cyl)
 	cyl->depth = gas_mod(cyl->gasmix, pO2, dive, 1);
 }
 
+cylinder_t create_new_cylinder(const struct dive *d)
+{
+	cylinder_t cyl = empty_cylinder;
+	fill_default_cylinder(d, &cyl);
+	cyl.start = cyl.type.workingpressure;
+	cyl.manually_added = true;
+	cyl.cylinder_use = OC_GAS;
+	return cyl;
+}
+
```

Comment:
this really should be in `equipment.c`, shouldn't it?
Let's move `fill_default_cylinder()` to the right file as well, instead of having these functions handing out in the planner...

Reply:
> this really should be in equipment.c, shouldn't it?

Yes, it should. The last time I wanted to move `fill_default_cylinder` there, I didn't because it takes a `dive *` argument and I couldn't decide if this is a dive-related or a cylinder-related function, so I left the declaration in `dive.h`. I guess `equipment.h` is more logical after all.

Reply:
Moved both functions to equipment.c